code/modules/mob/living/simple_animal/friendly/drone.dm

```diff
@@ -607,3 +608,21 @@
 	icon_state = "syndrone_item"
 	drone_type = /mob/living/simple_animal/drone/syndrone
 
+
+// malf drone
+/mob/living/simple_animal/drone/syndrone/malf
+	name = "Drone"
+	laws = \
+	"<span class='danger'>ERROR ER0RR $R0RRO$!R41.%%!!(%$^^__+ @#F0E4'STATION OVERRUN, ASSUME CONTROL TO CONTAIN OUTBREAK#*�&110010</span>\n"+\
+	"0. Keep the drone factory in good operating condition.\n"+\
+	"1. Intefere.\n"+\
+	"2. Kill.\n"+\
```

Comment:
Malf drones are servants of the AI, like borgs. At the very least, they should not have any potential to be hostile to it. Please update these laws to have the drones assist the AI or at the very least, not interfere with it or its borgs.